plugins/Marketplace/resources/qml/PackageCard.qml

```diff
@@ -354,28 +320,71 @@ Rectangle
                         onClicked: Qt.openUrlExternally(packageData.authorInfoUrl)
                     }
 
-                    Cura.SecondaryButton
+                    ManageButton
                     {
-                        id: disableButton
+                        id: enableManageButton
+                        state: packageData.stateManageEnableButton
                         Layout.alignment: Qt.AlignTop
-                        text: catalog.i18nc("@button", "Disable")
-                        visible: false  // not functional right now, also only when unfolding and required
+                        primaryText: catalog.i18nc("@button", "Enable")
+                        busyPrimaryText: catalog.i18nc("@button", "Enabling...")
+                        confirmedPrimaryText: catalog.i18nc("@button", "Enabled")
+                        secondaryText: catalog.i18nc("@button", "Disable")
+                        busySecondaryText: catalog.i18nc("@button", "Disabling...")
+                        confirmedSecondaryText: catalog.i18nc("@button", "Disabled")
+                        enabled: !(installManageButton.busy || updateManageButton.busy)
+                        visible: root.manageableInListView || root.expanded
+
+                        onClicked: {
+                            if (primary_action)
```

Comment:
Maybe we can just do `onClicked: packageData.setEnabledPackageTriggered(primary_action, packageData.packageId)`?

Not a requirement, just thought I'd mention the possibility.

Reply:
That is indeed a possibility but I'm curious at what the benefit might be? I'm inclined to leave it be If it's just a styling issue, except of course it is out of line with the rest of the style. If there are other benefits I'm more then willing to change it.